Dockerfile.dev

```diff
@@ -17,6 +17,7 @@ RUN \
         libcairo2 \
         gdb \
         git \
+        build-essential \
```

Comment:
Can you test it with the suggested change above @sirtux or @MariusMueller to only install the required dev dependencies instead of the full build-essential bundle ?

Reply:
Thank you @marcelveldt for following up on this pull request.

For my setup, you propoesed change works fine!